### DISCUSSION TITLE
Switch runtime from Alpine to Chimera

### DISCUSSION THREAD

**Discussion by Drsheppard01** [AppImage/AppImageKit#1376] [Ideas]
It is yet one experimental feature request

Why: Because mallocng from musl is underperfoming (see [tests](https://github.com/orgs/chimera-linux/discussions/2480))
Chimera uses mimalloc instead standard allocator

Benefits: potentially faster AppImages
Risks: Chimera is still alpha, and repo is poor

It still requires a lot of performance testing

**Comment by TheAssassin** [comment]
The influence of the runtime code itself is rather small overall, most of it comes from libfuse. I'd rather avoid using unstable distros and binaries, though.

**Reply by probonopd** [reply]
@VHSgunzo says [here](https://github.com/VHSgunzo/uruntime/issues/3#issuecomment-2559854123):

> I also tested AppImage type2-runtime (which is also now compiled on musl) and it too has a lower read speed than the squashfuse+glibc.

We should do some actual measurements before we jump to conclusions.

**Reply by probonopd** [reply]
Citing from over there:

> the difference in my test was ~15%
squashfuse+glibc vs type2-runtime

Thanks for the test @VHSgunzo.
